src/main/jhipster/src/main/webapp/app/ontology/variable-details/variable-details.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="modal-header" *ngIf="isModal">
+	<h5 class="modal-title">{{title}}</h5>
+	<button type="button" class="close" data-dismiss="modal" aria-label="Close" (click)="dismiss()">&times;
+	</button>
+</div>
+<div class="modal-body">
+	<nav>
+		<ul class="nav nav-tabs">
+			<li class="nav-item">
+				<a class="nav-link" jhiTranslate="ontology.variable-details.nav.detail"
+				   [routerLink]="['details']" [routerLinkActive]="['active']" queryParamsHandling="merge"></a>
+			</li>
+			<li class="nav-item">
+				<a class="nav-link" jhiTranslate="ontology.variable-details.nav.valid-values"
+				   [routerLink]="['valid-values/']" [routerLinkActive]="['active']" queryParamsHandling="merge"></a>
+			</li>
+		</ul>
+	</nav>
+	<div class="tab-content p-3">
+		<router-outlet></router-outlet>
+		<jhi-alert></jhi-alert>
+	</div>
+</div>
+<div class="modal-footer">
+	<button type="button" class="btn btn-primary" data-dismiss="modal" (click)="dismiss()">
+		<span jhiTranslate="release-notes.ok-button"></span>
```

Comment:
we create `global.ok`, we have some similar there already (_confirm, create, back, etc_)

Reply:
Done